### PR TITLE
chore!: update proto definitions

### PR DESCRIPTION
**Pull request by fryorcraken**
Use protobuf definitions from https://github.com/vacp2p/waku

Ref: https://github.com/waku-org/js-waku/issues/335

### CODE REVIEW

**Comment by fryorcraken** [comment]
Cc @LNSD @richard-ramos

File: packages/core/src/lib/store/index.ts [hunk 1]
```diff
@@ -408,7 +404,7 @@ export async function createCursor(
     digest,
     pubsubTopic,
     senderTime: messageTime,
-    receivedTime: messageTime,
+    receiverTime: messageTime,
```
Comment (fryorcraken) [inline packages/core/src/lib/store/index.ts:407]: I am surprised that using the waku message timestamp here works. Cc @LNSD @danisharora099

**Comment by richard-ramos** [comment]
@fryorcraken. would it make sense to instead of including the .proto files in the project, to use a git submodule pointing to vacp2p/waku?

File: packages/core/src/lib/filter/filter_rpc.ts [hunk 2]
```diff
@@ -8,16 +8,16 @@ export type ContentFilter = {
 /**
  * FilterRPC represents a message conforming to the Waku Filter protocol
  */
-export class FilterRPC {
-  public constructor(public proto: proto.FilterRPC) {}
+export class FilterRpc {
```
Comment (weboko) [inline packages/core/src/lib/filter/filter_rpc.ts:11]: Why did you change capitalisation here?

**Review by weboko — APPROVED** [review]

File: packages/core/src/lib/filter/filter_rpc.ts (hunk 2 above)
Comment (fryorcraken) [reply packages/core/src/lib/filter/filter_rpc.ts:11]: Capitalization was changed on proto file to proper PascalCase, I just match it as PascalCase is what we use across the codebase.

**Comment by fryorcraken** [comment]
> @fryorcraken. would it make sense to instead of including the .proto files in the project, to use a git submodule pointing to vacp2p/waku?

Maybe. There is an issue with the peer exchange definition (I need to create an issue) hence we cannot use the repo as is just yet.

The other question is whether `protons` can support proper protobuf module path, I don't know.

Let's review once peer exchange is done.

**Comment by fryorcraken** [comment]
> Maybe. There is an issue with the peer exchange definition (I need to create an issue) 

The issue: https://github.com/vacp2p/waku/issues/14
